README.Rmd

```diff
@@ -91,16 +89,15 @@ This function will produce the following structure and templates:
 Once you've created the project locally, initialize Git and push to github.com/FredHutch:
 
 ```{r eval=FALSE}
-library(usethis)
-
-use_git()
+usethis::use_git()
 
 # this will create a project on GitHub, push your work, and open the webpage!
 Sys.getenv("GITHUB_PAT")
-use_github(organisation = "FredHutch", private = TRUE, protocol = "https")
+usethis::use_github(organisation = "FredHutch", private = TRUE, protocol = "https")
+
 ```
 
-Note that you must have a [personal access token (PAT)](https://happygitwithr.com/github-pat.html) set up to do this.
+Note that you must have a [personal access token (PAT)](https://happygitwithr.com/github-pat.html) set up to do this. You can load in your PAT in every R session by editing your .Rprofile file (`usethis::edit_r_profile()`) and adding the line `Sys.setenv(GITHUB_PAT = "MY_PAT_HERE")` in the file.
```

Comment:
Taking a look at this again. This isn't specific to the VISCtemplates package so I am going to remove this sentence. It should go in our Git documentation [here](https://github.com/FredHutch/VISC-Documentation/blob/master/Programming/Git-Tips/01-git-github-training.md#set-up-a-github-personal-access-token). @seajane could you open a PR for the VISC-Documentation repo?